app/assets/images/arrow-down-fill.svg

```diff
@@ -1,3 +1,3 @@
-<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
-<path d="M12 16L6 10H18L12 16Z"/>
+<svg width="24" height="24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I don't think we want to remove `viewBox` since we'll need it if we ever decide to scale the SVG to fit a parent/container's space

Reply:
Great catch! I'm going to bring it back :D

Reply:
Correct. Same with all the other viewBox attributes removed below.